pallets/did/src/tests.rs

```diff
@@ -113,43 +104,460 @@ impl frame_system::Config for Test {
 
 impl Config for Test {
 	type Event = ();
-	type PublicSigningKey = H256;
-	type PublicBoxKey = H256;
 	type WeightInfo = ();
+	type DIDIdentifier = AccountId;
+}
+
+#[test]
+fn check_successful_did_creation() {
+	let did_identifier = AccountId::from([0u8; 32]);
+	let did_auth_keypair_seed = [1u8; 32];
+	let did_enc_keypair_public_bytes = [2u8; 32];
+	let pair = ed25519::Pair::from_seed(&*b"Alice                           ");
```

Comment:
I'd recommend to move accounts etc. which you use throughout all tests to the mock. Have a lock [here](https://github.com/KILTprotocol/mashnet-node/pull/129/files#diff-a9d1496400651f9fdde0dc064242adfd25968f4b6c3363e97218af4bc222e5c8R33) and [there](https://github.com/KILTprotocol/mashnet-node/pull/129/files#diff-5dd3697775b8b2a096d6d57fee37bb536f783f6d492650763ed9bc46e2f66bf8R29) for running a setup when building the test.

Reply:
As updating the tests require quite some work, I would personally move on to the other DID features, and then go back to this if and when we need to do it. But yes, I agree that there are better ways to implement the tests.

Reply:
It shouldn't require too much time IMO. If you want, I can take a look at this at some point. I wouldn't like to merge this as is right now due to the DID refactor being required for the first security audit. I know it's "just tests" but still.